content/_data/redirects.yaml

```diff
@@ -1,5 +1,34 @@
-- from: /open-source/accoutrement/
-  to: /accoutrement/
+- from: /community/
+  to: /birds/#sponsorships-and-donations
+- from: /speaking/
+  to: /services/speaking/
+- from: /work/services/
+  to: /services/
+- from: /services/design-systems-training/
+  to: /workshops/resilient-systems/
+
```

Comment:
I noticed on another page we went from /2015/08/14/coachhub-study/ to work/coachhub... is that something we would include in this list? I don't see this page on the current website so I'm not sure what was going on there.

Reply:
I guess we could -- it was a broken link on the old site, replaced by `/work/coachhub/`.